Automattic/jetpack
# Add editor plugin promoting Yoast collaboration

## Proposed changes:

* Add a new Jetpack + Yoast Jetpack plugin panel to all Jetpack sites.

<img width="308" alt="image" src="https://user-images.githubusercontent.com/426388/226837108-c6399cab-8e98-407d-a7af-af89a98591ad.png">

### Other information:


## Jetpack product discussion

* PT: pbNhbs-4KT-p2

## Does this pull request change what data or activity we track or use?

* No


## Testing instructions:

0. Start with a site that's connected to WordPress.com, or where you've enabled offline mode, with the following code snippet: `add_filter( 'jetpack_development_mode', '__return_true' );`
2. Enable Beta blocks on your site, with this code snippet: `add_filter( 'jetpack_blocks_variation', function () { return 'beta'; } );`
3. Go to Posts > Add New
4. You should see the panel in the Jetpack sidebar:

https://user-images.githubusercontent.com/426388/226837917-6db31733-b125-4c38-ad32-02f4e9ef294d.mov
